tests/unit/records/test_schemas.py

```diff
@@ -71,4 +73,42 @@ def test_contributors(app, db, minimal_record):
     minimal_record['contributors'] = [
         {'name': 'test', 'affiliation': 'test', 'type': 'Invalid'}
     ]
-    pytest.raises(ValidationError, Record.create, minimal_record)
+    exc = pytest.raises(ValidationError, Record.create, minimal_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+
+
+def test_full_json(app, db, full_record):
+    """Test full json."""
+    Record.create(full_record)
+
+
+def test_full_recid(app, db, full_record):
+    """Test recid property."""
+    full_record['recid'] = '12345'
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+
+
+def test_full_resource_type(app, db, full_record):
+    """Test resource type."""
+    full_record['resource_type'] = 'publication'
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+    full_record['resource_type'] = {'type': 'publication', 'subtype': 'book'}
+    Record.create(full_record)
+
+
+def test_full_contributors(app, db, full_record):
+    """Test contributors."""
+    full_record['contributors'] = [
+       {'affiliation': 'CERN', 'name': ' '', '' ', 'type': 'Other',
+                       'gnd': '', 'orcid': '0000-0002-1825-0097'}
+    ]
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
```

Comment:
@krzysztof .... i'm getting the "IOError: reading from stdin while output is captured"  while using ipdb